### ISSUE TITLE
Chart values are visible only with Hover and on-click

### ISSUE THREAD

**Issue by anukandimalla** [cityofaustin/homelessness-dashboard#200]
Description:
**User need:**

- The chart values need to be visible all the time along with hovering and on-click.
- Is there any specific reason for showing chart values in %?

**Why:**

- If the user is taking a printout or wants to understand the charts without interacting with the Dashboard charts, the associated data should be shown all the time.
- This update helps the users interacting dashboard on mobile devices and tablets.

### **Image**

![image.png](https://images.zenhubusercontent.com/5d236a251149c07fcf3f4fbc/1e18f552-4019-4552-9939-790a38472b5d)
Without Hover or on-click

![image.png](https://images.zenhubusercontent.com/5d236a251149c07fcf3f4fbc/b0dd550c-b4b4-4fd6-b48a-4d2d6dbf079b)
Tooltips are available on hover and on-click

### **Recommendation:**

![image.png](https://images.zenhubusercontent.com/5d236a251149c07fcf3f4fbc/39c6a374-0353-40be-bcf0-ee81aa8da04a)
To make it more understandable without any interaction or compatibility with printouts and mobile device

@toribr @ablangworthy27 and @sarahscottr

**Comment by toribr** [comment]
Good catch, @anukandimalla. I wonder if this is a setting we can change. Seems like showing the value for each bar would be an essential part of any library. @sarahscottr Is there a setting within tableau for this?

**Comment by toribr** [comment]
Actually @anukandimalla Is your primary concern with printing? On second thought, and depending on the chart type, adding values could essentially render the chart unreadable. I'm not sure we want to do that as a default.

**Comment by anukandimalla** [comment]
I am thinking it should be the default for **mobile devices** and **when users print the charts**- the aligned values will be helpful in reading the chart because the screen area is very less in mobile devices to open each tooltip to understand the value and also there is no possibility of interaction with screens/tooltips in the print version. 

**Not validated yet, just a thought** - The screenreader interaction behavior of the data tooltips.

I also understand each assign values creates readability issues and also concerns with data quality for showing them in numbers but we can ideate other alternatives to make the charts more readable.

**Comment by toribr** [comment]
Yes, I see. Some reading I have done suggests using tables, but I need to look into it further. The multiple axes per chart will complicate matters, too.
